data/keywords.yml

```diff
@@ -0,0 +1,248 @@
+- title: JSON Schema Keywords
```

Comment:
We dont need title of the page and description here ... it does not make sense. This file is only for keywords.